src/output.jl

```diff
@@ -106,10 +106,17 @@ end
 
 const _extension_map = Dict("tikz" => "tex")
 
-function addExtension(fn::AbstractString, ext::AbstractString)
+"""
+    addExtension(filepath, extension)
+
+Change filepath extension according to the extension map
+"""
+function addExtension(fp, ext::AbstractString)
+    parent = dirname(fp)
+    fn = basename(fp)
     oldfn, oldext = splitext(fn)
     oldext = chop(oldext, head = 1, tail = 0)
-    get(_extension_map, oldext, oldext) == ext ? fn : string(fn, ".", ext)
+    get(_extension_map, oldext, oldext) == ext ? fp : joinpath(parent, string(fn, ".", ext))
```

Comment:
Why is an absolute path needed  here ?

Reply:
Some package may work with absolute filepath no ?

Reply:
Yes, but it is a bit odd to force an absolute path in the second branch. Anyway I might be overthinking here.